yoti_test.go

```diff
@@ -464,7 +464,7 @@ func TestYotiClient_MissingPostalAddress_UsesFormattedAddress(t *testing.T) {
 	structuredPostalAddress, err = profile.StructuredPostalAddress()
 
 	assert.Assert(t, is.Nil(err))
-	assert.Equal(t, structuredPostalAddress.ContentType, yotiprotoattr.ContentType_JSON, "Retrieved attribute does not have the correct type")
+	assert.Equal(t, structuredPostalAddress.ContentType(), yotiprotoattr.ContentType_JSON.String(), "Retrieved attribute does not have the correct type")
```

Comment:
So there's no way we can prevent clients from having to add `()` to their code, if we also want to keep these values read-only, right?

Reply:
Yup.